material-overrides/assets/stylesheets/polkadot.css

```diff
@@ -5,6 +5,8 @@
   --athens-gray: #eceff3;
   --storm-gray: #6e7391;
   --black: #000000;
+  --code-green: #5ee20c;
+  --code-green-light: #64dd171a;
```

Comment:
I actually wouldn't use variables for these colors because they are not Polkadot-branded colors. So to avoid reusing these colors, please remove them from here. You can leave the `--md-code-admo-border-color` and `md-code-admo-title-background` if you want.